.gx/lastpubver

```diff
@@ -1 +1 @@
-0.1.19: QmXD921xzL9EDRpD6gRm1cb7Khm8VEpZ3NT3nPK7uTX6Fq
+0.1.19: QmXpK6aZZpAG38T6aSDBiYPU8wvDoHSczWmXurJzr82j4j
```

Comment:
You can use `gx release patch` instead of `gx publish -f` to release a new version.

Reply:
Does this happen after I merge?

Reply:
We can do it either after or before (as long as we don't merge anything else first). However, we shouldn't change the hash here until we bump the version number.

Reply:
How do I import gx hashes from elsewhere without publishing or bumping the version? Specifically https://github.com/libp2p/go-reuseport-transport/commit/143bb39150ed2e4c467ac3d6300a9121ee9e1767 in CI.

Reply:
You don't? I mean, you can do that for testing but I'd just release a new version (`gx release minor`).

Note: you'll need to *pin* the new hash. I've invited you to the pinbot IRC room.